courses/machine_learning/deepdive2/feature_engineering/labs/1_bqml_basic_feat_eng_bqml-lab.ipynb

```diff
@@ -21,7 +21,7 @@
     "\n",
```

Comment:
<p>Change to  TODO 3.  Although this is lab task #4, there are three objectives and TODO's are mapped to objectives.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Hi @gstripling , Updated the Lab TODOs according to three objectives given in the lab. Please review and feel free to merge the PR.

Thanks!